react/src/components/ComputeSessionNodeItems/SessionActionButtons.tsx

```diff
@@ -81,27 +92,53 @@ const SessionActionButtons: React.FC<SessionActionButtonsProps> = ({
   const userInfo = useCurrentUserInfo();
   const isOwner = userInfo[0]?.uuid === session?.user_id;
 
+  const hiddenButtons = React.useMemo(
+    () => new Set(hiddenButtonKeys ?? []),
+    [hiddenButtonKeys],
+  );
+  const isVisible = (key: SessionActionButtonKey) => {
+    // If system session, hide appLauncher, terminal, terminate buttons
+    if (
+      ['appLauncher', 'terminal', 'terminate'].includes(key) &&
```

Comment:
Shouldn't it be possible to terminate a system type session?

Reply:
Thank you I just fixed.